config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
+
   devise_for :users
-  get 'market_chirp/index'
+  get 'market_chirp/index', as: :market_chirp
```

Comment:
This changes the URL, and our market_chirp controller is our root controller.  We probably shouldn't do this.